soh/soh/Enhancements/Holiday/LL.h

```diff
@@ -0,0 +1,86 @@
+#ifndef LL_H
+#define LL_H
+
+#include "soh/Enhancements/cosmetics/CosmeticsEditor.h"
+#include "soh/Enhancements/game-interactor/GameInteractor.h"
+
+const size_t RAINBOW_PRESET_LEN = 9;
+
+static const char* RAINBOW_PRESET_NAMES[RAINBOW_PRESET_LEN] = {
+     "Christmas",
+     "Transgender",
+     "Nonbinary",
+     "Bisexual",
+     "Lesbian",
+     "Gay (MLM)",
+     "Asexual",
+     "Brazil",
+     "Italy"
+};
+
+static const ImVec4 RAINBOW_PRESETS[RAINBOW_PRESET_LEN][4] = {
+    { //christmas
+        {0.0/255.0, 140.0/255.0, 69.0/255.0, 0},
+        {205.0/255.0, 33.0/255.0, 42.0/255.0, 0},
+        {0.0/255.0, 140.0/255.0, 69.0/255.0, 0},
+        {205.0/255.0, 33.0/255.0, 42.0/255.0, 0}
+    },
+    { //trans
+        {255.0/255.0, 255.0/255.0, 255.0/255.0, 0},
+        {255.0/255.0, 159.0/255.0, 186.0/255.0, 0},
+        {255.0/255.0, 255.0/255.0, 255.0/255.0, 0},
+        {71.0/255.0, 186.0/255.0, 230.0/255.0, 0}
+    },
+
+    { //nb
```

Comment:
```suggestion
    { //enby
```
https://www.reddit.com/r/NonBinaryTalk/comments/yjndcb/theres_a_difference_between_nb_enby_and_nonbinary/